site/en/blog/fedcm-shipping/index.md

```diff
@@ -0,0 +1,179 @@
+date: 2022-11-09
+---
+
+The Federated Credential Management API (FedCM) is shipping in Chrome 108 (currently on the Beta channel). When it ships in Chrome Stable at the end of November 2022, the FedCM API will work in Chrome without requiring a flag or an origin trial token.
+
+Check the final API changes in the [accumulated update page](docs/privacy-sandbox/fedcm-updates).
+
+{% Aside %}
+
+If you are participating in the FedCM origin trial, please note that the trial is scheduled to end on November 22nd, which is slightly earlier than Chrome 108 ships.
+
+{% endAside %}
+
+
+Moving forward, we plan to introduce a number of new
+features based on the feedback we received from
+identity providers (IdP), relying parties (RP) and browser vendors. While we
+hope identity providers will adopt FedCM, please be aware that FedCM is still an
+API under active development and that backward incompatible changes are expected
+until Q4 2023.
+
+To minimize the challenges of deploying backwards incompatible changes, we
+currently have two recommendations for identity providers:
+
+-   Subscribe to our
+    [newsletter](https://groups.google.com/g/fedcm-developer-newsletter) where
+    we will provide updates as the API evolves.
+-   We strongly encourage IdPs to distribute the FedCM API via JavaScript
+    SDKs while the API is maturing, and to discourage RPs from self-hosting
+    SDKs. This will ensure IdPs can make changes as the API evolves, without
+    having to ask all of their relying parties to redeploy.
+
+## Background
+
+Over the last decade, identity federation has played a central role in raising
+the bar for authentication on the web, in terms of trustworthiness, ease-of-use
+(for example, passwordless single sign-in) and security (for example, improved
+resistance to phishing and credential stuffing attacks) compared to per-site
+usernames and passwords.
+
+Unfortunately, the mechanisms that identity federation has relied on (iframes,
+redirects and cookies) are actively being abused to track users across the web.
+As the user agent isn't able to differentiate between identity federation and
+tracking, the mitigations for the various types of abuse make the deployment of
+identity federation more difficult.
+
+[The Federated Credential Management API (FedCM)](https://fedidcg.github.io/FedCM/)
+provides a use-case-specific abstraction for federated identity flows on the
+web, by exposing a browser mediated dialog that allows users to choose accounts
+from IdPs to login to websites.
+
+FedCM is a multi-step journey to make identity on the web better, and in its
+first step we are focused on reducing the impact of third-party cookie phase-out
+on federated identity (see below for a few [steps
+further](#heading=h.hd4vvssat3s0)).
+
+
+
+{% Video src="video/vgdbNJBYHma2o62ZqYmcnkq3j0o1/TJLjWp1nVLlDMMCK2ugQ.mov", autoplay="true", loop="true" %}
+
+
+Chrome has been
+[experimenting with FedCM since Chrome 101](/blog/fedcm-origin-trial/).
```

Comment:
As we start redirecting this URL, I'd replace this with something else, maybe:

```suggestion
[experimenting with FedCM since Chrome 101](/origintrials/#/register_trial/3977804370874990593).
```